packages/metadata-common/src/AddMetadataButton.tsx

```diff
@@ -90,11 +90,7 @@ export const AddMetadataButton = (
               ? (): void => props.addMetadata(props.schemas?.[0].name)
               : handleToggle
           }
-          title={`Create new ${
-            singleSchema
-              ? props.schemas?.[0].display_name
-              : props.schemas?.[0].schemaspace
-          }`}
+          title={`Create new ${props.schemaType}`}
```

Comment:
But here, it seems like `schemaType` is the full title.  

For single and multi-schema schemaspaces, what are examples of `schemaType`'s value?

Reply:
This is the hover text on the + button to add a new metadata from our metadata widgets.
With elyra current schemaspaces, for single schema, we have
```
Create new code snippet
Create new runtime image
```
and for multiple schemas:
```
Create new runtime configuration
Create new component catalog
```